AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m

```diff
@@ -30,6 +30,19 @@
 #import <PINCache/PINCache.h>
 
 #if PIN_ANIMATED_AVAILABLE
+@interface ASPINRemoteImageManager : PINRemoteImageManager
```

Comment:
ASPINRemoteImageManager` is declared inside `PIN_ANIMATED_AVAILABLE` but you use it also if it’s not defined in line 120 I think